src/app/services/wizard.service.ts

```diff
@@ -80,7 +78,26 @@ export class WizardService {
     service.fetchProjectDetails(url).subscribe(project => {
       this.fetchedProject.next(project);
       this.router.navigate([this.addManualProjectRoute]);
-    });
+    },
+    () => {
+      const alertConfig: AlertConfig = {
+        type: AlertType.info,
+        preMessage: null,
+        mainMessage: 'We were unable to automagically retrieve information about your project. You can manually add this information using the form!',
+        dismissible: true,
+        timeout: this.alertService.defaultTimeout
+      };
+      const emptyProject: MappedProject = {
+        collaborators: [],
+        name: '',
+        description: '',
+        shortDescription: '',
+        uri: url
+      };
+      this.fetchedProject.next(emptyProject);
```

Comment:
Why set a emptyProject if we could also just redirect to the manual component? I don't see any added value of setting the fetchedProject to and empty project.